src/reducers/deeplink.js

```diff
@@ -1,13 +1,16 @@
 import {
   SET_DEEPLINK_DATA,
-    SET_DEEPLINK_URL
+    SET_DEEPLINK_URL,
+    SET_DEEPLINK_DATA_EXTRAPARAMS
   } from "../utils/constants/storeType";
   
   export const deeplink = (
     state = {
       url: null,
       data: {},
-      id: null
+      id: null,
+      fromService: null,
+      extraParams: null
```

Comment:
After reading through the code, I understand why this is necessary, seeing as data is reserved for data parsed from the deeplink itself, and in some cases we need data to be passed through the deeplink reducer, but could you please rename it to "passthrough" so its more clear what purpose it serves, and then renaming the "fqn" parameter you put here to automatically bring up the link identity to "fqnToAutolink" or something along those lines to also make it clear that putting data there will automatically prompt the user to link the id when the appropriate screen is reached.